packages/expo-updates/android/build.gradle

```diff
@@ -87,6 +87,10 @@ if (!safeExtGet("expoProvidesDefaultConfig", false)) {
 }
 
 android {
+  buildFeatures {
```

Comment:
Could you explain why these Gradle changes were needed?

Reply:
Best I can explain is that when building just the module we need to also build the config since we use `BuildConfig` in the module code. Previously, this was built by default. It is also set in the root app gradle build script so normal app building works.

Concretely, trying to run the integration tests was failing due to this not being set.

Here's an explanation: https://medium.com/androiddevelopers/5-ways-to-prepare-your-app-build-for-android-studio-flamingo-release-da34616bb946